test/user.test.js

```diff
@@ -991,7 +976,19 @@ describe('User', function() {
           if (err) return done(err);
 
           var errorResponse = res.body.error;
-          assert.equal(errorResponse.code, 'LOGIN_FAILED_EMAIL_NOT_VERIFIED');
+
+          // extracting code and details error response
+          let errorExcerpts = {
```

Comment:
no... already tried before fall-backing to this trick.
your proposal does not work with nested objects and leads to 
` Uncaught AssertionError: expected { Object (statusCode, name, ...) } to have a property 'details' of { Object (userId) }, but got { Object (userId) }`